include/merlin/core_kernels.cuh

```diff
@@ -556,6 +566,9 @@ __global__ void rehash_kernel_for_fast_mode(
               new_bkt_idx, start_idx, buckets_size, bucket_max_size,
               buckets_num, table->dim);
           if (rank == 0) {
+            K hashed_key = Murmur3HashDevice(static_cast<K>(EMPTY_KEY));
+            uint8_t target_digest = static_cast<uint8_t>(hashed_key >> 32);
+            bucket->digests[key_idx] = target_digest;
```

Comment:
For `EMPTY_KEY`, we'd better define a separate macro for its relative digest.

Reply:
Ok, thanks. Maybe a force inline device function is needed, and do you mind if I do it later in another PR for optimizing insert operation?

Reply:
I get, and I will precompute its value and use it directly